config.lua

```diff
@@ -732,13 +732,13 @@ lspconfig.texlab = add_lsp {
 }
 
 ---# typescript-language-server
---- __Status__: Untested
+--- __Status__: Works
 --- __Site__: https://github.com/typescript-language-server/typescript-language-server
 --- __Installation__: `npm install -g typescript-language-server typescript`
 lspconfig.tsserver = add_lsp {
   name = "typescript-language-server",
   language = "javascript",
-  file_patterns = { "%.jsx?$", "%.cjs$", "%.mjs$", "%.tsx?$" },
+  file_patterns = { "%.js?$", "%.jsx?$", "%.cjs$", "%.mjs$", "%.tsx?$" },
```

Comment:
No need for this change.

Reply:
Right, the `?` symbol.

Reply:
I mean that `"%.jsx?$"` already includes `.js` files.